config/bootscripts/boot-edge.cmd

```diff
@@ -0,0 +1,45 @@
+# DO NOT EDIT THIS FILE
+#
+# Please edit /boot/armbianEnv.txt to set supported parameters
+#
+
+setenv load_addr "0x39000000"
+setenv overlay_error "false"
+# default values
+setenv rootdev "/dev/mmcblk0p1"
+setenv verbosity "1"
+setenv console "display"
```

Comment:
Why not "both"?
Having possibility to use UART console to boot and log into OS is beneficial and should be enabled by default IMHO.